app/controllers/application_controller.rb

```diff
@@ -47,14 +47,33 @@ def deobfuscate(hash)
   end
 
   def show_notification_count
-    if current_user
-      @count = TaskManagement.
-               notifications_count(current_user.user_type, current_user.id)
-    end
+    @count = Notification.unseen(current_user).count if current_user
```

Comment:
Why do we have these methods in the application controller? Controllers are meant to be kept slim.